source/AI.cpp

```diff
@@ -3437,7 +3442,32 @@ void AI::AimTurrets(const Ship &ship, FireCommand &command, bool opportunistic)
 				}
 
 				// Determine how much the turret must turn to face that vector.
-				double degrees = (Angle(p) - aim).Degrees();
+				double degrees = 0.0;
```

Comment:
```suggestion
				double degrees = 0.;
```
Not sure if we need this 0 at all since we're assigning a value in the if..else block below.